docs/reference-guide/query-modules/implement-custom-query-modules/api/mock-python-api.md

```diff
@@ -0,0 +1,93 @@
+---
+id: mock-python-api
+title: Mock Python query module API
+sidebar_label: Mock Python API
+slug: /reference-guide/query-modules/api/mock-python-api
+---
+
+The mock Python query module API enables you to develop and test query modules
+for Memgraph without having to run a Memgraph instance by simulating its
+behavior. As the mock API is compatible with the
+[Python API](/reference-guide/query-modules/implement-custom-query-modules/api/python-api.md),
+you can add modules developed with it to Memgraph as-is, without modifying the
+code.
+
+It is implemented in `mgp_mock.py`, which contains definitions of all
+classes and functions provided for developing query module procedures and
+functions. The source file is found in the Memgraph installation directory,
```

Comment:
The source file can be found

or

The source file is located <- i like this one better

Reply:
Done